zds/tutorialv2/views/views_published.py

```diff
@@ -125,6 +125,7 @@ def get_context_data(self, **kwargs):
                                           if reaction.author == self.request.user]
 
         context['isantispam'] = self.object.antispam()
+        context['pm_link'] = self.object.get_absolute_contact_url(u'À propos de')
```

Comment:
Ok, merci pour l'essai.

Le 04/04/2016 11:31, Anto59290 a écrit :
>
> In zds/tutorialv2/views/views_published.py 
> <https://github.com/zestedesavoir/zds-site/pull/3486#discussion_r58349548>:
>
> > @@ -125,6 +125,7 @@ def get_context_data(self, **kwargs):
> >                                            if reaction.author == self.request.user]
> >
> >          context['isantispam'] = self.object.antispam()
> > +        context['pm_link'] = self.object.get_absolute_contact_url(u'À propos de')
>
> Je viens d'essayer et mon titre de Mp se transforme en :
>
>     django.utils.functional.*proxy* object at 0xae1e7c8c - essai
>
> a la place de :
> A propos de - essai
>
> Aucune raison de pourquoi. Il me semble que dans la fonction, le 
> paramètre par defaut n'est pas internationalisé :
> https://github.com/zestedesavoir/zds-site/blob/dev/zds/tutorialv2/models/models_database.py#L182
>
> Je suis preneur d'explications si quelqu'un a une idée
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub 
> <https://github.com/zestedesavoir/zds-site/pull/3486/files/548127f45f6b04900d620d87a9a0a17e4de8a402#r58349548>
>